src/osd/PGTransaction.h

```diff
@@ -20,6 +20,7 @@
 
 #include "common/hobject.h"
 #include "osd/osd_types.h"
+#include "osd/osd_internal_types.h"
```

Comment:
It was just the first (and apparently only) place I hit a build error and added it as there weren't any further errors after that.